src/fragments/lib/project-setup/ios/create-application/20_install_libraries.mdx

```diff
@@ -2,17 +2,15 @@
 
 <Block name="Swift Package Manager">
 
-To start adding the Amplify Libraries to your iOS project, open your project in Xcode and select **File > Swift Packages > Add Package Dependency**.
+To start adding the Amplify Libraries to your iOS project, open your project in Xcode and select **File > Add Packages...**
 
 ![Add package dependency](/images/project-setup/20_4_add-package-dependency.png)
 
-Enter the Amplify iOS GitHub repo URL (`https://github.com/aws-amplify/amplify-ios`) into the search bar and click **Next**.
+Enter the Amplify iOS GitHub repo URL (`https://github.com/aws-amplify/amplify-ios`) into the search bar and wait for the result to load.
 
-![Search for repo](/images/project-setup/20_5_search-amplify-repo.png)
-
-You'll see the Amplify iOS repository rules for which version of Amplify you want Swift Package Manager to install. Choose the first rule, **Version**, as it will use the latest compatible version of the dependency that can be detected from the `main` branch, then click **Next**.
+You'll see the Amplify iOS repository rules for which version of Amplify you want Swift Package Manager to install. Choose the dependency rule as , **Up to Next Major Version**, as it will use the latest compatible version of the dependency that can be detected from the `main` branch, then click **Add Package**.
```

Comment:
```suggestion
You'll see the Amplify iOS repository rules for which version of Amplify you want Swift Package Manager to install. Choose the dependency rule **Up to Next Major Version**, as it will use the latest compatible version of the dependency that can be detected from the `main` branch, then click **Add Package**.
```